src/js/components/profile-form/profile-form.html

```diff
@@ -482,31 +482,31 @@
 										ng-model="profile.assignments[0].mcc"
 										ng-required="isRequired( 'mcc' )">
 									<option value="Student-Led Movements" translate>Student-Led Movements</option>
-									<option value="Leader-Led Movements" translate>Leader-Led Movements</option>
-									<option value="Global Church Movements" translate>Church-Led Movements</option>
+									<option value="Leader-Led Movements" translate>LeaderImpact</option>
+                                                                        <option value="Global Church Movements" translate>GCM</option> <!-- MK: obfuscate value as well? -->
```

Comment:
If we want to change these, and I'd suggest we don't, It would involve updating enum values in Global Registry as well as any other systems code that pushes these into or reads from Global Registry.